integration/test/alert/alert.go

```diff
@@ -29,9 +29,25 @@ var vserverArwState = []string{
 	`"dry-run"`,
 }
 
-type PromAlert struct {
-	message string
-	count   int
+var hmAlertState = []string{
+	`"RaidLeftBehindAggrAlert"`,
+	`"RaidLeftBehindSpareAlert"`,
+	`"NoISLPresentAlert"`,
+	`"ClusterSeveredAllLinksAlert"`,
+	`"StorageBridgePortDown_Alert"`,
+	`"StorageBridgeTempAboveCritical_Alert"`,
+	`"StorageBridgeTempBelowCritical_Alert"`,
+	`"StorageBridgeUnreachable_Alert"`,
+	`"StorageFCAdapterFault_Alert"`,
+	`"InterconnectAdapterOfflineAlert"`,
+	`"RaidDegradedMirrorAggrAlert"`,
+	`"InterclusterBrokenConnectionAlert"`,
+	`"SASAdapterOffline_Alert"`,
+	`"FabricSwitchFanFail_Alert"`,
+	`"FabricSwitchPowerFail_Alert"`,
+	`"FabricSwitchTempCritical_Alert"`,
+	`"FabricSwitchTempSensorFailed_Alert"`,
+	`"FabricSwitchUnreachable_Alert"`,
```

Comment:
There would be some improvement in ci code to remove these, That I will do post release.